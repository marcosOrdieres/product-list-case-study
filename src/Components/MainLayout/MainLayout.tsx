import styled from 'styled-components';

export interface MainLayoutProps {
  children?: any;
}

const MainLayoutStyled = styled.div`
  display: flex;
  flex-direction:column;
  width: 100vw;
  height: 100vh;
  align-items: center;
  padding-bottom:50px;
  margin-bottom:50px;
`;

export const MainLayout: React.FC<MainLayoutProps> = (props) => <MainLayoutStyled {...props}>{props.children}</MainLayoutStyled>;

export default MainLayoutStyled;