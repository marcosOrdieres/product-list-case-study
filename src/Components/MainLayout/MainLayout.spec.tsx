import { render } from '@testing-library/react';
import React from 'react';

import MainLayout from './MainLayout';

describe('MainLayout', () => {
  it('should render and match styles', () => {
    const { asFragment } = render(<MainLayout>MainLayout</MainLayout>);
    expect(asFragment()).toMatchSnapshot();
  });
});
