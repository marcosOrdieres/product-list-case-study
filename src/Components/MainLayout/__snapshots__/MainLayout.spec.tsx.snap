// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainLayout should render and match styles 1`] = `
<DocumentFragment>
  <div
    class="sc-bdfBQB alQAU"
  >
    MainLayout
  </div>
</DocumentFragment>
`;
